src/types.ts

```diff
@@ -75,9 +75,9 @@ export type Signer = ethers.Signer & TypedDataSigner;
 
 /** Return type for any on chain call */
 export interface ContractMethods {
-  call: (additionalOverrides?: Overrides) => Promise<ContractTransaction>;
-  estimateGas: (additionalOverrides?: Overrides) => Promise<BigNumber>;
-  callStatic: (additionalOverrides?: Overrides) => Promise<any>;
+  call: (additionalOverrides?: Overrides | PayableOverrides) => Promise<ContractTransaction>;
+  estimateGas: (additionalOverrides?: Overrides | PayableOverrides) => Promise<BigNumber>;
+  callStatic: (additionalOverrides?: Overrides | PayableOverrides) => Promise<any>;
```

Comment:
Here you can replace `Overrides` with `PayableOverrides` because PayableOverrides is defined like this:

```
interface PayableOverrides extends Overrides {
    value?: BigNumberish | Promise<BigNumberish>;
}```